test/unit/ngsiv2/plugins/alias-plugin_test.js

```diff
@@ -144,6 +180,239 @@ describe('Attribute alias plugin', function() {
                 .reply(204);
         });
 
+        it('should rename the attributes as expected by the mappings', function(done) {
+            iotAgentLib.update('light1', 'Light', '', values, function(error) {
+                should.not.exist(error);
+                contextBrokerMock.done();
+                done();
+            });
+        });
+    });
+    describe('When an update comes for attributes with aliases and integer type', function() {
+        var values = [
+            {
+                name: 'ut',
+                type: 'Number',
+                value: '99823423'
+            }
+        ];
+
+        beforeEach(function() {
+            nock.cleanAll();
+
+            contextBrokerMock = nock('http://192.168.1.1:1026')
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', 'gardens')
+                .post('/v2/entities/light1/attrs', utils.readExampleFile(
+                    './test/unit/ngsiv2/examples/contextRequests/updateContextAliasPlugin3.json'))
+                .reply(204);
+        });
+
+        it('should rename the attributes as expected by the mappings', function(done) {
+            iotAgentLib.update('light1', 'Light', '', values, function(error) {
+                should.not.exist(error);
+                contextBrokerMock.done();
+                done();
+            });
+        });
+    });
+
+    describe('When an update comes for attributes with aliases and integer type.' + 
+        'The IOTA sends the update with alias and using JSON native type', function() {
```

Comment:
This "The IOTA sends the update with alias and using JSON native type" fragment is not used in any other describe()... I wonder if this is intentional or a leftover of a previous refactor.

Reply:
Fixed in 3208e030ec9701f6ceb353b9c3586c9d6cbdc8fc